src/RestClient.ts

```diff
@@ -1254,8 +1253,16 @@ export class RestClient extends BaseRestClient {
    */
   submitSpotBatchOrders(
     params: SpotOrder[],
+    xGateExptime?: number,
```

Comment:
As you see here, I cannot put it directly in array, but it is same as in the docs. Separated, and then later handled to include header.